recipes/newrelic/infrastructure/alerts/golden.yml

```diff
@@ -21,6 +21,34 @@ preInstall:
       - High Application Response Time
       - Low Application Throughput
       This policy can be modified under Alerts & AI\Policies.
+  requireAtDiscovery: |
+      if [[ "$NR_CLI_SHOW_GOLDEN_OPTION" == "true" ]] ; then exit 0; fi 
```

Comment:
@NRhzhao are we missing a line before that like this:
NR_CLI_SHOW_GOLDEN_OPTION="{{.NR_CLI_SHOW_GOLDEN_OPTION}}"

Reply:
@Julien4218  Hi Julien, I belive {{.NR_CLI_SHOW_GOLDEN_OPTION}} is part of go-task.
The pre-install is not part of go-task, and it's not getting the variable with that notation.

It just use the regular $ notation.

Reply:
Makes sense, thanks @NRhzhao I will test. I kind of think we should also have a task at some point to make the preInstall script work the same (support the {{. }} templating, but that may not be a small effort), so we'll see.